test/multiple_dcs_SUITE.erl

```diff
@@ -66,7 +66,7 @@ init_per_testcase(_Case, Config) ->
 end_per_testcase(_, _) ->
```

Comment:
Clean up in this file!
Remove comments and check whitespace, such that only minimal changes are made.
Also, please check this for *all* other files.